assets/controller.yaml

```diff
@@ -284,12 +284,10 @@ spec:
                 path: clouds.yaml
         - name: config-cinderplugin
           configMap:
-            name: openstack-cinder-config
+            name: csi-cinder-config
```

Comment:
How about we call this CM `cloud-conf`and store the file under the `cloud.conf` key, to align with the CCM's cloud config?